### ISSUE TITLE
Edit command in Finance page needs better description for amount field

### ISSUE THREAD

**Issue by itssisi** [itssisi/pe#11]
The amount field only specifies that only positive numbers can be added. I was a bit confused as I thought that it means that I can't change a positive commission to become an expense. Only after I tested the product then I realised that `amount` refers to only the numbers and not commission or expense. It would be better to explain it in the UG that changing the `amount` does not allow users to change commission into expenses and vice versa. You can highlight that `amount` only changes the numerical value instead of the + and -. 

![Screenshot 2023-11-17 at 5.26.43 PM.png](https://raw.githubusercontent.com/itssisi/pe/main/files/b6003667-0dbd-4bce-83ba-1bfedffd57fd.png)

**Comment by nus-pe-script** [comment]
# Team's Response
Not in scope since it is still technically right to say that the format only allows positive numbers up to two decimal points (i.e., if you input `edit 1 a/-200` it will throw invalid command error). 


But we will make it clearer by using your suggestion in further specifying that amount only changes the numerical value instead of the + and -, thanks! :>
# Items for the Tester to Verify
## :question: Issue response

Team chose [`response.NotInScope`]

- [ ] I disagree

**Reason for disagreement:** [replace this with your explanation]

<catcher-end-of-segment><hr>
## :question: Issue severity

Team chose [`severity.Low`]
Originally [`severity.Medium`]

- [ ] I disagree

**Reason for disagreement:** [replace this with your explanation]

<catcher-end-of-segment><hr>
